crates/polars-ops/src/chunked_array/strings/escape_regex.rs

```diff
@@ -0,0 +1,16 @@
+use polars_core::prelude::arity::unary_elementwise;
+use polars_core::prelude::StringChunked;
+use regex::escape;
+
+#[inline]
+pub fn escape_regex_str(s: &str) -> String {
```

Comment:
Added this function so `pl.escape_regex` and `str.escape_regex` will be coupled by same implementation.